drivers/gpio/gpio-starfive-jh7100.c

```diff
@@ -0,0 +1,545 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * GPIO driver for StarFive JH7100 SoC
+ *
+ * Copyright (C) 2020 Shanghai StarFive Technology Co., Ltd.
+ */
+
+#include <linux/module.h>
+#include <linux/gpio/driver.h>
```

Comment:
It seems we need `#include <linux/interrupt.h>` for compile-testing on other architectures.